common/src/main/java/ru/coderedwolf/wordlearn/common/ui/BaseFragment.kt

```diff
@@ -15,14 +16,20 @@ import moxy.MvpAppCompatFragment
 import ru.coderedwolf.wordlearn.common.di.ComponentManager.clearInjector
 import ru.coderedwolf.wordlearn.common.di.ComponentManager.inject
 import ru.coderedwolf.wordlearn.common.di.generateComponentName
+import ru.coderedwolf.wordlearn.common.util.ContextExtensionsHolder
 
 const val STATE_COMPONENT_NAME = "state_component_name"
 
-abstract class BaseFragment : MvpAppCompatFragment() {
-    protected abstract val layoutRes: Int
-    private lateinit var fragmentComponentName: String
+abstract class BaseFragment : MvpAppCompatFragment(), ContextExtensionsHolder {
+
+    private var fragmentComponentName: String = ""
     private var instanceStateSaved: Boolean = false
 
+    override val extensionContext: Context
+        get() = requireContext()
```

Comment:
wtf?

Reply:
It's need for context extension interface

Reply:
Because we can use extension everywhere there is context